controllers/front/OrderController.php

```diff
@@ -237,6 +237,25 @@ public function displayAjaxselectDeliveryOption()
         ]));
     }
 
+    public function displayAjaxcheckProductsStillOrderable(): void
+    {
+        $responseData = [
+            'allGood' => true,
+            'cartUrl' => '',
+        ];
+
+        if ($this->context->cart->isAllProductsInStock() !== true ||
```

Comment:
Maybe it would be good to have these checks in a dedicated method?

Reply:
@kpodemski Made an issue for this so we don't forget ;-) https://github.com/PrestaShop/PrestaShop/issues/27023